backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -819,7 +780,7 @@ export class UserResolver {
       logger.debug('PrivateKey encrypted...')
 
       // Save new password hash and newly encrypted private key
-      userEntity.password = newPasswordHash[0].readBigUInt64LE()
+      userEntity.password = encryptPassword(userEntity, passwordNew)
```

Comment:
you are missing to set the password type?

`userEntity.passwordEncryptionType = PasswordEncryptionType.GRADIDO_ID`

Reply:
you are right